alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/util/AlertingUtilsTests.kt

```diff
@@ -0,0 +1,60 @@
+package com.amazon.opendistroforelasticsearch.alerting.util
```

Comment:
Missing license header

Reply:
Addressed in https://github.com/opendistro-for-elasticsearch/alerting/pull/360